src/store/HawkState.ts

```diff
@@ -287,11 +287,13 @@ export function useHawkState(initialSearch?: Partial<Request>): [SearchStore, Se
 			doHistory = true;
 		}
 
+		const removeSearchParams = (config.removeSearchParams && fromInput);
```

Comment:
Would be great if you add a note above this line.

Reply:
Done